Data/AppData/I_DataText.csv

```diff
@@ -50,7 +50,7 @@ Jan 2023-Dec 2023"," ","<a href='https://www.nomisweb.co.uk/datasets/apsnew'>Ann
 <li>Each estimate from the Annual Population Survey carries a margin of error. These are available in the original data via NOMIS. Large margins of error are usually associated with groups with only a small number of respondents. Therefore, please take caution when interpreting data from small subgroups.</li>
 <li>Use caution when interpreting this data. A difference between subgroups does not necessarily imply any causality. There could be other contributing factors at work.</li>
 </ol>","Inactivity","are inactivity volumes changing","Inactivity in","Inactivity","inactivity volumes","inactivity volume","Inactivity is"
-"vacancies","May 2023 data","These statistics should be treated as official statistics in development (previously known as experimental statistics). Further data will be published in summer 2024.","<a href='https://www.ons.gov.uk/employmentandlabourmarket/peopleinwork/employmentandemployeetypes/datasets/labourdemandvolumesbystandardoccupationclassificationsoc2020uk'>ONS Textkernel</a>"," ","<ol>
+"vacancies","Dec 2023 data","These statistics should be treated as official statistics in development (previously known as experimental statistics). Further data will be published in late summer 2024.","<a href='https://www.ons.gov.uk/employmentandlabourmarket/peopleinwork/employmentandemployeetypes/datasets/labourdemandvolumesbystandardoccupationclassificationsoc2020uk'>ONS Textkernel</a>"," ","<ol>
```

Comment:
Is late summer confirmed - should we have that in the data sources table?

Reply:
This is based on plans that have been communicated to Drew but the date hasn't been announced by ONS so I suggest we don't put a specific date in the data sources table.